lib/new.js

```diff
@@ -174,47 +183,73 @@ const cloneRepo = () => {
  * Prompts for the project's root repo URL.
  * @name promptForUrl
  */
-const promptForUrl = () => {
-  const prompt = 'What is the URL to your repo? (leave this blank if you don\'t know)';
-  return promptly.prompt(prompt, { 'default': '' })
-    .then((value) => {
-      settings.url = value;
-      return Promise.resolve();
-    });
+const promptForUrl = (args) => {
+  const message = 'What is the URL to your repo? (leave this blank if you don\'t know)';
+  const save = (url) => {
+    settings.url = url;
+    return Promise.resolve();
+  };
+
+  if (args.repo === false) {
+    return save('');
+  }
+
+  if (args.repo) {
+    return save(args.repo);
+  }
+
+  return promptly.prompt(message, { 'default': '' })
+    .then((value) => save(value));
 };
 
 /**
  * Prompts for the project's root directory name.
  * @name promptForName
  */
-const promptForName = () => {
-  const prompt = 'What is the root directory for your SPA? (example: my-spa-name)';
+const promptForName = (args) => {
   const validator = (value) => {
     const packageName = getPackageName(value);
 
     if (!value || !value.match(/^[a-z0-9\-]*$/)) {
-      throw new Error(
+      logger.error(
         'SPA root directories may only contain lower-case letters, numbers or dashes.'
       );
+      throw new Error();
```

Comment:
Also, my console printed the following:
```
$ skyux new --name FOOBAR
SKY UX processing command new
SPA root directories may only contain lower-case letters, numbers or dashes.
ERROR MESSAGE
```
It also proceeded to the next step of the process instead of exiting. Is that accurate?

Reply:
The `ERROR MESSAGE` was a rogue `console.log` hanging out.  I'll remove that silly thing.

When you say "proceeded to the next step," can you confirm you see "What is the root directory for your SPA? (example: my-spa-name)" displayed. That's actually the first step, which is bypassed by passing in the `name` argument.